devito/passes/iet/openmp.py

```diff
@@ -230,6 +232,12 @@ class Ompizer(object):
     Coefficient to adjust the chunk size in parallelized non-affine Iterations.
     """
 
+    DYNAMIC_WORK = 10
```

Comment:
A CS individual counting in base 10, would have expected 8 or 16 ;)

Reply:
lolz